pkg/operator/edgefs/cluster/mgr/mgr.go

```diff
@@ -247,6 +252,20 @@ func (c *Cluster) makeUiService(name string) *v1.Service {
 	}
 
 	k8sutil.SetOwnerRef(c.context.Clientset, c.Namespace, &svc.ObjectMeta, &c.ownerRef)
+
+	if c.dashboardSpec.LocalAddr != "" {
+		ip := net.ParseIP(c.dashboardSpec.LocalAddr)
+		if ip == nil {
+			logger.Errorf("wrong dashboard localAddr format")
+			return svc
+		}
+
+		if !ip.IsUnspecified() {
```

Comment:
Just wondering do you think it would be good to log if the IP is "unspecified"?

Reply:
not a bad idea! Done!